packages/commerce-sdk-react/package.json

```diff
@@ -0,0 +1,54 @@
+{
+    "name": "commerce-sdk-react",
+    "version": "2.2.0-dev",
+    "description": "A library that provides react hooks for fetching data from Commerce Cloud",
+    "author": "dev@mobify.com",
+    "license": "See license in LICENSE",
+    "engines": {
+      "node": "^14.0.0",
+      "npm": "^6.14.4 || ^7.0.0 || ^8.0.0"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/SalesforceCommerceCloud/pwa-kit.git"
+      },
+      "homepage": "https://github.com/SalesforceCommerceCloud/pwa-kit/tree/develop/packages/ecom-react-hooks#readme",
+      "bugs": {
+        "url": "https://github.com/SalesforceCommerceCloud/pwa-kit/issues"
+      },
+    "scripts": {
+      "build": "cross-env NODE_ENV=production internal-lib-build build && tsc --emitDeclarationOnly",
+      "build:watch": "watch 'npm run build' src/",
+      "lint": "npm run lint:js && tsc --noEmit",
+      "lint:fix": "npm run lint:js -- --fix",
+      "lint:js": "internal-lib-build lint \"**/*.{js,jsx,ts,tsx}\"",
+      "test": "internal-lib-build test",
+      "test:watch": "npm test -- --watch",
+      "test:inspect": "node --inspect-brk jest --runInBand",
+      "typecheck": "tsc --noEmit",
+      "version": "node ./scripts/version.js",
+      "format": "internal-lib-build format \"**/*.{js,jsx,ts,tsx}\""
+    },
+    "main": "dist/index.js",
+    "types": "dist/index.d.ts",
+    "files": [
+      "dist"
+    ],
+    "private": true,
```

Comment:
This means that the package will not be published on NPM. https://docs.npmjs.com/cli/v7/configuring-npm/package-json#private